src/go/networking/pkg/portproxy/server_test.go

```diff
@@ -217,7 +217,10 @@ func marshalAndSend(listener net.Listener, portMapping types.PortMapping) error
 	if err != nil {
 		return err
 	}
-	c, err := net.Dial(listener.Addr().Network(), listener.Addr().String())
+	testDialer := net.Dialer{
+		Timeout: 5 * time.Second,
+	}
+	c, err := testDialer.DialContext(context.Background(), listener.Addr().Network(), listener.Addr().String())
```

Comment:
Oh, we don't pass in a context here do we? We probably should… it's just this test file.